rest/common/utils/rest.py

```diff
@@ -0,0 +1,339 @@
+"""
+Python module for initiating and executing commands via REST API.
+"""
+
+import json
+import time
+
+import requests
+from dotenv import dotenv_values
+
+from rest.common.config.config import Config
+from rest.common.utils.exceptions import CommandExecutionError, HTTPError
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def rest():
+    """
+    REST method to be used by any test method
+    """
+    config = dotenv_values(
+        ".env"
+    )  # config = {"IP": "10.12.34.23", "USERNAME": "admin", "PASSWORD":"passwd", "PORT":"3211"}
```

Comment:
please remove this commented code.

Reply:
This comment is added to understand how config is read from .env file. Please read the description of the PR and I am giving demo of design choices and tradeoffs tomorrow, if interested kindly join

Reply:
done